api/helpers/validation/customerValidation.js

```diff
@@ -1,9 +1,33 @@
 const { RequestValidationError } = require('@helpers/errors');
 
 class CustomerValidation {
-    static SensownaNazwa(customerData) {
+    static CustomerNameError(customerData) {
         if (!customerData.name || !/\S/.test(customerData.name)) {
-            throw new RequestValidationError('Name can\'t be empy');
+            throw new RequestValidationError('Name can\'t be empty');
+        }
+    }
+
+    static CustomerIdError(customerId) {
+        if (isNaN(customerId)) {
+            throw new ArgumentIsNotIntError('Customer ID must be an integer. Given ID: ' + customerId);
+        }
+    }
+
+    static CustomerUpdateError(updatedCustomerData) {
+        if (updatedCustomerData === null || updatedCustomerData === undefined) {
+            throw new Error('Provide values to update.');
```

Comment:
add custom errors to helpers/errors.js so that they can provide more information about the error